operator/src/test/java/org/keycloak/operator/testsuite/integration/BaseOperatorTest.java

```diff
@@ -175,6 +174,9 @@ private static void createRBACresourcesAndOperatorDeployment() throws FileNotFou
             ((ClusterRoleBinding)obj).getSubjects().forEach(s -> s.setNamespace(namespace));
         } else if (obj instanceof RoleBinding && "keycloak-operator-view".equals(((RoleBinding)obj).getMetadata().getName())) {
             return null; // exclude this role since it's not present in olm
+        } else if (obj instanceof Deployment) {
+            // set values useful for testing - TODO: could drive this in some way from the test/resource/application.properties
```

Comment:
Now that you mention application.properties. It might be quite easy to get it, no? SmallRye Config is available in the test context, isn't it?

This is not a blocker, though. Approving regardless.

Reply:
> It might be quite easy to get it, no? SmallRye Config is available in the test context, isn't it?

Easy to get to the config or even the properties file - not so easy to determine what should be set on the operator as we put other stuff in there for unit testing.